### PR TITLE
Updates form to Powergate signup.

### PR DESCRIPTION
**Pull request by johnnymatthews**

### CODE REVIEW

File: docs/store/lotus/store-data.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ t016309
 ```
 
 ::: callout
-If you want to use these miners please [fill in this Google form](https://forms.gle/f5Vd5kTNYTKrmj1D8) including information from our daemon (`lotus wallet list` and `lotus net id`). You will be contacted via Slack/email when your Lotus Node has been whitelisted.
+If you want to use these miners please [fill in this Google form](https://blog.textile.io/hosted-powergate//f5Vd5kTNYTKrmj1D8) including information from our daemon (`lotus wallet list` and `lotus net id`). You will be contacted via Slack/email when your Lotus Node has been whitelisted.
```
Comment (pooja) [inline docs/store/lotus/store-data.md:93]: I think this form doesn't need to be changed, just the first one!

**Review by pooja — APPROVED** [review]
One change, but otherwise lgtm!

File: docs/store/lotus/store-data.md (hunk 1 above)
Comment (johnnymatthews) [reply docs/store/lotus/store-data.md:93]: Cool beans! I'll take it out.

Comment (johnnymatthews) [inline docs/store/lotus/store-data.md:93]:
```suggestion
If you want to use these miners please [fill in this Google form](https://forms.gle/f5Vd5kTNYTKrmj1D8) including information from our daemon (`lotus wallet list` and `lotus net id`). You will be contacted via Slack/email when your Lotus Node has been whitelisted.
```

**Pull request merged**